qd_proto.lua

```diff
@@ -39,6 +41,21 @@ function qd_proto.prefs_changed()
     end
 end
 
+-- Parse QD message.
```

Comment:
The function description must be in present tense. `Parses the QD message`.
https://www.doxygen.nl/manual/commands.html#cmdparam
Parameter names should begin with the definite article.